master/internal/experiment.go

```diff
@@ -108,7 +108,7 @@ type (
 func newExperiment(master *Master, expModel *model.Experiment, taskSpec *tasks.TaskSpec) (
 	*experiment, error,
 ) {
-	conf := expModel.Config
+	conf := &expModel.Config
 
 	// Validate the ResourcePool setting.  The reason to do it now and not in postExperiment like
 	// all the other validations is that the resource pool should be revalidated every time the
```

Comment:
Can you delete this comment?  It isn't my code that was buggy; it was my understanding of what the old code did (as this comment explains).

Side-note, now that I understand correctly what the behavior is, there is a large cleanup that can be done.  For example, all the other default filling happens in a `parseCreateExperiment`, and this should too.

(but I'm fine landing this fix now, since it is correct as-written, and I'll do the followup fix in the next few days)